content/ember-released-6-8.md

```diff
@@ -0,0 +1,165 @@
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 6.8
+
+Ember.js 6.8 is an incremental, backwards compatible release of Ember with bug fixes, performance improvements, and minor deprecations.
+
+v6.8.0 of `ember-source` is the first minor version of the package to be published to `npm` with [Trusted Publishing](https://docs.npmjs.com/trusted-publishers). We will be implementing this across all our packages.
+
+At the bottom of the [npm package page](https://www.npmjs.com/package/ember-source), you'll find a section labeled 'Provenance' that provides verification that the package contents were published from the source repository.
+
+#### Bug fixes
+
+Ember.js 6.8 introduces 1 bug fix.
+
+- Removes unnecessary package ember-cli-htmlbars-inline-precompile from component-test blueprint
+
+#### Features
+
+Ember.js 6.8 introduces 2 new features.
+
+##### `renderComponent`
+
+The new `renderComponent` API provides a powerful way to render components into any DOM element, making it easier to integrate components in other environments like d3, ag-grid, WYSIWYG editors, and more! This feature is particularly useful for micro applications, REPLs, and "islands"-based tools.
+
+`renderComponent` can be imported from `@ember/renderer` and accepts a component definition along with configuration options:
+
+```gjs
+import { renderComponent } from '@ember/renderer';
+
+const Greeting = <template>Hello {{@name}}!</template>;
+
+const result = renderComponent(Greeting, {
+  into: document.querySelector('#my-element'),
+  args: { name: 'World' }
+});
+
+// Clean up when done
+result.destroy();
+```
+
+The API supports several configuration options including:
+
+- `into`: The DOM element to render into
+- `args`: Arguments to pass to the component - these can be a `trackedObject`
+- `owner`: Optional owner object for service access, (or minimal partial implementation of what your component needs)
+
+[Live Demo](https://limber.glimdown.com/edit?c=JYWwDg9gTgLgBAbzlApgOwCYqgYQuCNdeAXzgDMp84ByAARRACNsB6VTbbGgbgChQkWIjgwoAQwDGAaxQYA8kwBWKSaQpUQtBszaopMYADcUrSRAA2F1YcIBnXnz7m0d%2BG-EwUcALyiJMnKKKmoAFEh8cHDmAK5oMABccAAMADRwkXDAaJKhAJQiHl4AdLHxANTlcGR8JHn8zvbwAGIQEL5wADxe4BaeKAB8mZ3kwCgWGHYoMAOd1gDm6BgDrRCdrAtLQ1FRCAhFKKUQcTAkJJlRnUwxMDCEcISSFsAyPnsHxdmSZyttlevXW6EbZdVijcaTaZDdY9MB9LwDBrkOJqYD3DhYKD5RCZazwcaMYgdDAQSQxEDEUr6LwAUWsFPioRoGGMNHqmQx2DwBCIjNW6SQ2TuSQJDNI7KiqBgMSgaDgouI-HOfG6jDh-RBAGUYP04OJMMgUAZjMAYABPPWoOB2AAW4lQGASTku4ImUxmcxQi0wAwASm0YOtNj6Loh9jqSmVTucdl1AXc5Y9nq93hHDl8fqt-qx48DhmCxm6oZlMntQpyoHkzirWLD4YMgA&format=gjs&shadowdom=on)
```

Comment:
we have never linked to limber from a release blog before and we should not be starting to do so here. This should stand on its own as a doc or link to guides or docs for the thing, not limber